packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -62,7 +63,7 @@ const getCustomerStepStatus = createSelector(
             return {
                 type: CheckoutStepType.Customer,
                 isActive: false,
-                isComplete: customer?.isStripeLinkAuthenticated !== undefined ?? isComplete,
+                isComplete: paymentProviderCustomer?.authenticationState !== undefined,
```

Comment:
Why do you remove `?? isComplete` part?

Reply:
Nick pointed in comment above that `isComplet` is not needed here

Reply:
oh yeah, I see. I believe it should be something like `isComplete || paymentProviderCustomer?.authenticationState`
But I'm not sure about it

Reply:
What would be the value of `paymentProviderCustomer?.authenticationState` for guest checkout when the customer step is done?
I saw its definition here -> https://github.com/bigcommerce/checkout-sdk-js/blob/2f8f6d88f6f92b046a6517e6fcca5ba5b84cf092/packages/core/src/payment-provider-customer/payment-provider-customer.ts#L7.

Reply:
@peng  authenticationState for Stripe is added in this PR
https://github.com/bigcommerce/checkout-sdk-js/pull/2193/files#diff-4bd826524519b1e1bf0df4c1a87b9d868efbc298b96e4cd86c6c345a2e2946b3R14 which is checkout-sdk part of the same task, but it is not merged yet.
For guest after customer step is done, `paymentProviderCustomer?.authenticationState` is:

- `false` if Stripe Link user is not authenticated
- `true` if Stripe Link user is authenticated

Reply:
However I am not sure what is Braintree behaviour. https://github.com/bigcommerce/checkout-sdk-js/blob/2f8f6d88f6f92b046a6517e6fcca5ba5b84cf092/packages/core/src/payment-provider-customer/payment-provider-customer.ts#L7 says that it can be string or undefined.

I am guessing that the idea of 
 `isComplete: customer?.isStripeLinkAuthenticated !== undefined ?? isComplete`
was to check if `customer?.isStripeLinkAuthenticated` is `undefined` and if yes then return `isComplete`.
But `customer?.isStripeLinkAuthenticated !== undefined` was never `undefined`. It was always true or false. Because of this 
`isComplete` was always set to `true` or `false`, never to `isComplete`.

So maybe if this was working until now, we should keep it like this, with removed `?? isComplete` part as it was never used:
`isComplete: paymentProviderCustomer?.authenticationState !== undefined`

Reply:
> @peng authenticationState for Stripe is added in this PR https://github.com/bigcommerce/checkout-sdk-js/pull/2193/files#diff-4bd826524519b1e1bf0df4c1a87b9d868efbc298b96e4cd86c6c345a2e2946b3R14 which is checkout-sdk part of the same task, but it is not merged yet. For guest after customer step is done, `paymentProviderCustomer?.authenticationState` is:
> 
> * `false` if Stripe Link user is not authenticated
> * `true` if Stripe Link user is authenticated

You @ wrong person! Please @ right person